docs/faq/groups.md

```diff
@@ -10,51 +10,58 @@ description: Player Grouping Functionality in Music Assistant
     
 ## Manual Sync
 
-These are temporary group players that are configured via the [Player List](ui.md#player-list).
+These are temporary group players that are configured via the [Player List](ui.md#player-list) or HA action. They provide a very flexible way to create and destroy groups as the players can be added or removed at any time. 
 
 ### Features
 
-- Players can dynamically be added or removed before or during playback
-- The HA action [media_player.join](https://www.home-assistant.io/integrations/media_player/#action-media_playerjoin) can be used to create this type of group
+- Players can be dynamically added or removed before or during playback
+- Players in the group will play in perfect sync
+- The HA actions of [media_player.join](https://www.home-assistant.io/integrations/media_player/#action-media_playerjoin) and [media_player.unjoin](https://www.home-assistant.io/integrations/media_player/#action-media_playerunjoin) can be used to manage this type of group
 
 ### Known Issues / Notes
 
-- Players can be linked via the chain icon in the [Player List](../ui.md#player-list)
-- Only like player types that support synced playback are allowed to be joined. These players will then play in sync
+- Players can be linked via the checkbox in the [Player List](../ui.md#player-list)
+- Only like player types that support synced playback are allowed to be joined
 - When players are grouped in this way only the main player holds the queue. For example, player A has an existing queue and players B and C are joined to player A. If player A is turned off players B and C will stop
-- Some player types may pause momentarily as a new device is joined 
+- Some player types may pause momentarily as a new device is joined
+- HA automations could be used to quickly create an often used group of players 
 
 ## Sync Groups
```

Comment:
```suggestion
## Group Players
```

Reply:
This section is specifically about Sync Groups. The next section is about Universal?